fixieai/__init__.py

```diff
@@ -3,6 +3,8 @@
 from fixieai.agents import CodeShotAgent
 from fixieai.agents import Embed
 from fixieai.agents import Message
+from fixieai.agents import OAuthParams
+from fixieai.agents import RunHelper
```

Comment:
`RunHelper` isn't super-intuitive to me. Maybe something like `Context` or `Agent` to carry this info?

Reply:
Per our discussion, I changed the API so that Funcs can select what arguments do they want to accept through the argument names.
I think it looks pretty neat :) I think we can support this in most interpreter-based languages.

Reply:
It's either this or request.query, request.user_storage, request.oauth_handler. Let's try this and see how it goes.